### PR TITLE
112 minimal rounding on points and custom data (and calculations thereof)

### PR DESCRIPTION
**Pull request by mmcdermott**
This adds a higher precision setting for point estimates and custom data entries, which persist through calculation. 
varying precision

### CODE REVIEW

File: src/lib/guesstimator/formatter/formatters/DistributionTextUpTo.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const item = Object.assign(
     format(g) {
       const guesstimateType = this.guesstimateType(g)
       const [low, high] = this._numbers(g.text)
-      return {guesstimateType, low, high }
```
Comment (OAGr) [inline src/lib/guesstimator/formatter/formatters/DistributionTextUpTo.js]: I find this quite strange.  If someone estimates, [0.000001,0.00002], would that return as much precision as [5,7]?

File: src/lib/guesstimator/formatter/formatters/DistributionPointText.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ export const item = Object.assign(
     format(g) {
       const {guesstimateType} = this
       const value = parseNumber(g.text)
-      return {guesstimateType, value}
```
Comment (OAGr) [inline src/lib/guesstimator/formatter/formatters/DistributionPointText.js]: So if someone enters 0, this would return '0.0000'?

**Comment by OAGr** [comment]
Let's talk about this tomorrow.

File: src/lib/guesstimator/formatter/formatters/DistributionTextUpTo.js (hunk 1 above)
Comment (mmcdermott) [inline src/lib/guesstimator/formatter/formatters/DistributionTextUpTo.js]: Precision is probably poorly named, this just specifies the number of nonzero digits in scientific notation to show for types of estimates. So this actually matches current behavior afaict (e.g. [0.00001, 0.00002] has the same # of nonzero digits as [5,7])

File: src/lib/guesstimator/formatter/formatters/DistributionPointText.js (hunk 2 above)
Comment (mmcdermott) [inline src/lib/guesstimator/formatter/formatters/DistributionPointText.js]: No, it renders as 0. I modified something else that makes it strip trailing zeros from the representation in all cases.

**Comment by mmcdermott** [comment]
Sounds good. This may have been attacking a different issue than intended here; all this PR does is make it so that stats about fixed inputs render with more digits shown in the final representation. It doesn't actually do any real precision/sig-fig propagation.

**Comment by OAGr** [comment]
Let's refactor this to be simpler.

**Pull request closed without merging**